scheduler/tests.py

```diff
@@ -52,7 +52,7 @@ class RepeatableJobFactory(BaseJobFactory):
     result_ttl = None
     interval = 1
     interval_unit = 'hours'
-    repeat = None
+    repeat = 0
```

Comment:
Changing this meaning changing the behavior of all tests for `RepeatableJobFactory`.
Please revert this.

Reply:
Done, Thanks!